packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -864,14 +864,20 @@ class _PlatformViewLinkState extends State<PlatformViewLink> {
 
   @override
   Widget build(BuildContext context) {
-    if (_controller == null) {
+    final PlatformViewController? controller = _controller;
+    if (controller == null) {
       return const SizedBox.expand();
     }
     if (!_platformViewCreated) {
-      // Depending on the platform, the initial size can be used to size the platform view.
-      return _PlatformViewPlaceHolder(onLayout: (Size size) => _controller!.create(size: size));
+      // Depending on the implementation, the initial size can be used to size
+      // the platform view.
+      return _PlatformViewPlaceHolder(onLayout: (Size size) {
+        if (controller.awaitingCreation) {
```

Comment:
This change (and the new `awaitCreation` logic) is the core of the fix. Hybrid Composition was pretty broken because plugins call `create` is called in the initial view build, per our docs, and if `onLayout` here fired before that async operation was completed (which is when _platformViewCreated will actually change, down in `_onPlatformViewCreated`), then it would start a *second* call to create.

TLHC and VD weren't affected by this because the initial `create` doesn't have a size, and thus in those implementations doesn't do anything.